personal.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal</title>
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+</head>
+
+
+<header>
+    <img src="https://upload.wikimedia.org/wikipedia/commons/6/61/Deadmau5_d.jpg" alt="Deadmau5">
+    <h1>Deadmau5 Personal Section</h1>
+    <nav>
+        <div><strong>#</strong></div>
+         <div><a href="index.html">Executive Summary</a></div>
```

Comment:
You and I have different wording here, but we can figure that out with the group.